sylius/b2b-kit/0.1/manifest.json

```diff
@@ -0,0 +1,15 @@
+{
+    "aliases": ["b2b-kit"],
+    "bundles": {
+        "Sylius\\B2BKit\\SyliusB2BKitPlugin::class": ["all"],
+        "OldSound\\RabbitMqBundle\\OldSoundRabbitMqBundle::class": ["all"],
+        "BitBag\\SyliusWishlistPlugin\\BitBagSyliusWishlistPlugin::class": ["all"],
+        "FOS\\ElasticaBundle\\FOSElasticaBundle::class": ["all"],
+        "BitBag\\SyliusElasticsearchPlugin\\BitBagSyliusElasticsearchPlugin::class": ["all"],
+        "FOS\\JsRoutingBundle\\FOSJsRoutingBundle::class": ["all"],
+        "Sylius\\OnePageCheckoutPlugin\\SyliusOnePageCheckoutPlugin::class": ["all"]
```

Comment:
```suggestion
```

Reply:
Aaaaah you're right. I'm committing your suggestion @oallain :)